### DISCUSSION TITLE
v4.1.0 - Disable Asset Tag from showing up next to Device Name

### DISCUSSION THREAD

**Discussion by MattDRogers** [netbox-community/netbox#17416] [Q&A] [answered]
Hello everyone. I recently upgraded to version 4.1.0. One thing that's throwing me off is that wherever a Device Name shows up, the Asset Tag value is always stuck next to it in parenthesis. Like this: SERVER-01-A(TAG-4303).

This manifests at the top of device pages, in drop lists, and in cable connections on device interface pages. It's not something that's of particular value to my organization, eats up finite line real estate and is redundant at best.

Is there a way to toggle this feature off? I've looked in the Official Docs, the Discussion Boards and Issues, but I can't find anyone calling this out. Any pointers would be very helpful. Thanks.

Examples (no "real" hostnames):
![Screenshot 2024-09-09 092416](https://github.com/user-attachments/assets/0b56da02-f508-4322-a00e-e47c0ba8e3bb)
![Screenshot 2024-09-09 092549](https://github.com/user-attachments/assets/dd8f9491-c83d-4e56-be21-976f7da5c70a)
![Screenshot 2024-09-09 092744](https://github.com/user-attachments/assets/7fd1acdb-aa62-45f4-bbb4-a65f22a4076f)

**Comment by ghost** [comment, marked as answer]
You'll need to raise a feature request.

Workaround might be to move the asset tag to a custom field.

**Comment by MattDRogers** [comment]
Submitted:
[FR-17425](https://github.com/netbox-community/netbox/issues/17425)

**Comment by MattDRogers** [comment]
The Feature Request is kind of stalled out one week later.
Does anybody else utilize the Asset Tags field and would like the option to not have it stuck onto the Device Name everywhere in the UI? 
Also, I can't seem to find any record or justification for this change being implemented in any of the code release notes.
I could be missing it, but I searched for "asset tag" and came up with nothing.

**Reply by ghost** [reply]
I absolutely would prefer the asset tag is not seen everywhere in the UI. only on the object.
